api/core/v1alpha2/virtual_machine.go

```diff
@@ -408,3 +408,8 @@ const (
 	ProvisioningTypeUserDataRef ProvisioningType = "UserDataRef"
 	ProvisioningTypeSysprepRef  ProvisioningType = "SysprepRef"
 )
+
+const (
+	SecretTypeCloudInit corev1.SecretType = "provisioning.virtualization.deckhouse.io/cloud-init"
+	SecretTypeSysprep   corev1.SecretType = "provisioning.virtualization.deckhouse.io/sysprep"
```

Comment:
It seems very important to reflect this in the documentation, such as docs/EXAMPLES.md, docs/FAQ.md, and so on.

Reply:
Not reflected